tree/ntuple/v7/doc/BinaryFormatSpecification.md

```diff
@@ -802,6 +802,27 @@ This section is a comprehensive list of the C++ types with RNTuple I/O support.
 Within the supported type system complex types can be freely composed,
 e.g. `std::vector<MyEvent>` or `std::vector<std::vector<float>>`.
 
+### Type Name Normalization
+
+Type names are stored according to the following normalization rules
+  - All whitespaces are removed.
```

Comment:
Not exact phrasing, in part because of https://github.com/root-project/root/pull/17637/files#r1943506781

Reply:
Clarified.